index.html

```diff
@@ -16,9 +16,14 @@
 <body>
     <section class="main-page">
         <header class="header">
-            <a href="" class="logo"><h3>Ali Ahmad Karimi</h3></a>
+            <a href="#" class="logo"><h3>Ali Ahmad Karimi</h3></a>
             <nav class="toolbar">
-                <img src="images/Icon-Menu.png" alt="Menu Icon">
+                <img src="images/Icon-Menu.png" alt="Menu Icon" class="img-3">
+                <ul>
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
             </nav>
         </header>
        <section class="context">
```

Comment:
- [ ] `optional`
- [ ] Kindly the social icons as the figma template please the screenshots below
`figma template`
![Screenshot from 2022-07-07 10-17-14](https://user-images.githubusercontent.com/66230306/177875203-d8eb1929-a2df-4a10-8177-f6783e9170a5.png)

`your template`
![Screenshot from 2022-07-07 10-36-39](https://user-images.githubusercontent.com/66230306/177875509-25741258-be31-4050-9db2-52ab9f54ab2a.png)